public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -42,19 +42,31 @@ export interface State {
   timeInfo?: string;
   timeRange?: TimeRange;
   errorMessage: string | null;
+
+  // Current state of all events
+  data: PanelData;
 }
 
 export class PanelChrome extends PureComponent<Props, State> {
   timeSrv: TimeSrv = getTimeSrv();
+  queryRunner = new PanelQueryRunner();
+  querySubscription: Unsubscribable;
+  queryWidthPixels = -1; // used to set maxDataPoints in query
```

Comment:
and interval :)

Reply:
I don't follow... interval gets calculated later?